andela/thor-ah
# #159987675 integrate eslint with project

#### What does this PR do?
Configure ESLint for the project

#### Description of Task to be completed?
Configure ESLint to make use of the airbnb styleguide

#### How should this be manually tested?
- Clone the repo
- cd into the ‘thor-ah’ directory
- Make sure you have eslint installed globally
- Run ’npm install’
- Run ‘npm run lint

#### What are the relevant pivotal tracker stories?
[159987675](https://www.pivotaltracker.com/story/show/159987675)
